plugins/JSAPIExample/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 set(TARGET_NAME JSAPIExample)
 setup_hifi_client_server_plugin(scripting)
-link_hifi_libraries(shared plugins)
+link_hifi_libraries(shared plugins script-engine)
+include_hifi_library_headers(networking)
```

Comment:
There's an include-demand because it's including script-engine and I wasn't able to get all the network #includes out of the .h files.  I did try to keep them down as little as possible (there used to be a *lot* more in that list nearer to the beginning).  It's not a link-demand in any case.